.gitignore

```diff
@@ -5,3 +5,4 @@
 /tmp
 .DS_Store
 /test/e2e/assets/simple-app/empty-dir
+dist/
```

Comment:
should we move this into tmp/dist?

Reply:
hmm... I don't see a config for goreleaser to make this change. 
but also, goreleaser will practically only ever be run by an ephemeral env (github action). So the `dist` folder should rarely (if ever) make it on a dev workstation.